<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Genetic architecture of tropical maize for interaction with plant growth-promoting bacteria via high-throughput phenotyping</title>
    <meta charset="utf-8" />
    <meta name="author" content="PhD Student: Rafael M. Yassue   Advisor: Roberto Fritsche Neto" />
    <meta name="date" content="2020-02-11" />
    <link href="libs/remark-css-0.0.1/default.css" rel="stylesheet" />
    <link href="libs/remark-css-0.0.1/default-fonts.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Genetic architecture of tropical maize for interaction with plant growth-promoting bacteria via high-throughput phenotyping
### PhD Student: Rafael M. Yassue <br> Advisor: Roberto Fritsche Neto
### Genetics and Plant Breeding Program
### 02/11/2020

---


#Rafael Massahiro Yassue
&lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt;
![](rafael.png) 

---
#  Summary 
  + Introduction
    - Maize
    - Nitrogen 
    - Holobiont
    - Plant Growth-Promoting Bacteria
    - ‚ÄúBreeding pillars‚Äù and Phenomics
    - Components of electromagnetic spectrum
    - Plant Growth-Promoting Bacteria and HTTP
  + Goals
  + Material and methods
    - Pilot trial 
  + Results 
    - Orthomosaic
    - Genotyping-by-sequencing
    - Population structure
    - Population parameters
  + Outlook
  


---
#Maize 
![](mundo.png)

---
#Nitrogen
- Average maize crop removes 160 kg/ha of nitrogen &lt;br&gt;

--

- BNF by PGPB has been reported to contribute up to 12‚Äì33% of total N uptake in maize (Monta√±ez et al., 2009)&lt;br&gt;

--

- PGPR may be a good alternative to more sustainable production

--

- 300 maize-associated endophytes (mostly bacteria) &lt;br&gt;

--

- Based on the genomes currently in NCBI, bacteria and fungi contain an average of &lt;b&gt; 3,800 and 9,600 genes &lt;/b&gt; , respectively. If we assume &lt;b&gt; 35 unique fungi and 175 &lt;/b&gt;  unique bacteria per plant (see Sect. 14.2.4), it implies that the maize endosphere contains  &lt;b&gt; 858,000‚Äì1.14 million genes &lt;/b&gt; , over twenty times that of the maize plant itself (Hardoim et al., 2015). &lt;br&gt;


---
#Holobiont
![](holob.png)
--
![](eq.png)

---
#Plant Growth-Promoting Bacteria

![](plos1png.png)

---
#Plant Growth-Promoting Bacteria

![](domes.png)

---
#Plant Growth-Promoting Bacteria
 &lt;img src="effects.png" width="1000" height="350" /&gt; 

---
#Plant Growth-Promoting Bacteria
 
&lt;img src="effects2.png" width="1000" height="350"  /&gt; 

---
#Plant Growth-Promoting Bacteria
 &lt;img src="effects3.png" width="1000" height="350"  /&gt; 

---
#Plant Growth-Promoting Bacteria

- Type of Soil (Oliveira et al., 2006) &lt;br&gt;

--

- Genotype x PGPB  (Monta√±ez et al., 2009) &lt;br&gt;

--

- Environment interaction (Rouphael et al., 2018) &lt;br&gt;

--

- Nitrogen disponibility (Egamberdiyeva, 2007) &lt;br&gt;

--

- Microbiome x microbiome interaction (Van Der Heijden et al., 2008)

--

- Heterosis (Vidotti et al. 2019, preprint Wagner et al. 2020)

---
# ‚ÄúBreeding pillars‚Äù and Phenomics
![](pheno.png)
---
# ‚ÄúBreeding pillars‚Äù and Phenomics
![](pheno2.png)
---
#Components of electromagnetic spectrum
![](espec.png)
--
&lt;img src="table1.png" style="width: 60%" /&gt;


---
#Plant Growth-Promoting Bacteria

&lt;img src="microbioma.png" style="width: 100%" /&gt;

---
#Plant Growth-Promoting Bacteria
&lt;img src="microb.png" style="width: 100%" /&gt;
---
#Plant Growth-Promoting Bacteria and HTTP
&lt;img src="HTTP2.png" style="width: 100%" /&gt;
---
#Plant Growth-Promoting Bacteria and HTTP
&lt;img src="HTTP.png" style="width: 100%" /&gt;
---
#Goal
- Genetic and phenotypic characterization of maize inbred lines for response to plant growth-promoting bacteria

--

- Implementation of a High-Throughput Phenotyping platform in the greenhouse

--

- Prospecting new traits that may be associated with a positive response to plant x PGPB interaction

---
#Material and methods

- Greenhouse condition

--

- 360 inbred lines, with and without PGPB

--

- V2, V4, and V6 

| Classic phenotyping | High-throughput phenotyping  |
| ------------- | ------------- |
| Plant height (3x) | NDVI (3x) |
| Stalk diameter | NDVIred (3x) |
| Number of leaves (3x) | NDVIgreen (3x) |
| Chlorophyll content (3x) | Chlorophyll indices (3x) |
| Canopy mass | Canopy temperature (3x) |
| Dry mass of canopy and root | Plant height (3x) |
| | WinRHIZO | 


---
# Material and methods
Table. Blend of Bacterias 

|Bacteria	|Mechanism|
|------|-------|
|&lt;i&gt;Bacillus &lt;/i&gt;sp. RZ2MZ9 |	P solubilization, ACC deaminase activity and ammonia and IAA production|
|&lt;i&gt; Azospirillum brasilense &lt;/i&gt; Ab-v5 	|Fixing N2 and synthesis of phytohormones|
|&lt;i&gt; Pantoea agglomerans &lt;/i&gt; 33.1	|Synthesize indoleacetic acid and solubilize phosphate|
|&lt;i&gt; Delftia &lt;/i&gt; sp. 93A. 	|High nitrogen-fixing activity and acetylene reduction|

---
# Material and methods

- Augmented blocks (Federer), 6 blocks with 60 lines and 3 checks with and without PGPB (I)
  \begin{aligned}
y=X 1 \boldsymbol{g}+X 2 \boldsymbol{b}+X 3 \boldsymbol{c}+X 4 \boldsymbol{p}+X 5 \boldsymbol{i}+\boldsymbol{\varepsilon}
  \end{aligned}
  
--


- Sown three times (II)
  \begin{aligned}
y=x 1 g+X 2 e+X 3 p+X 4 i+\varepsilon
  \end{aligned}

g effects of genotype &lt;br&gt;
b effects of blocks  &lt;br&gt;
c effects of checks  &lt;br&gt;
p effect of inoculation with PGPB  &lt;br&gt;
e effect experiment effect of  &lt;br&gt;
i effect of interaction genotype x inoculation with the PGPB   &lt;br&gt;
ùú∫ ~ùëÅ(0, ùëπ).

---
# Pilot trial


- Type of soil

--

- Microbiome

--

- Fertilization ‚Äì Hoagland without Nitrogen

--

- Data collection pipeline and image processing ‚Äì Metashape and QGIS


---
# Pilot trial
&lt;br&gt;
.pull-left[![](thermal.gif)]
.pull-right[![](RGB2.gif)]
&lt;br&gt; &lt;br&gt;
.middle[Figure. Collection of thermal (left) and multispectral (right) images]


---

# Orthomosaic 

.pull-bottom[.center[![](orto.png)]]
Figure. Orthomosaic using multispectral camera

---
# Genotyping-by-sequencing 



.pull-left[ - SNP: 14639 &lt;br&gt;  - Two enzymes approach - Pst1-MSE1 (Poland, 2011) &lt;br&gt; - SNPCalling: TASSEL 5.0 &lt;br&gt;  - Aligner: Bowtie 2 &lt;br&gt; - Imputation: Beagle 5.0
&lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; ]
--

.pull-right[ &lt;img src="omics.png" style="width: 250%" /&gt; ]

 
.bottom[ &lt;font size="3"&gt; Figure. Raw data coverage (A), and depth (B), Filtered data coverage (C), and depth (D) &lt;/font&gt; ]

---
# Population structure

.pull-left[![](NI.jpeg)]
.pull-right[![](PCA.gif)]
&lt;br&gt;
.middle[Figure. Proportion of variation accumulated (A) and Cumulative proportion of variance explained by the principal components (1, 2 and 3, B)]

---

# Population structure

&lt;img src="STRUTpng.png" style="width: 100%" /&gt; &lt;br&gt;
Figure. Genotypes sorted by origin (A) and group (K) (B)


---
# Population structure

![](gagd.png)
Figure. Additive (Ga), and dominance (Gd) relationship matrix
---
# Population parameters 
|Parameter	|Esalq	|Iapar	|All |
| ------------- | ------|----- |-----|
|Ne	|101.50	        |98.82	        |199.83|
|Va	|4385.2	|4422.9	|4493.14|
|Vd	|1635.88	|1635.27	|1644.02|
|Number of fixed alleles	|30	|80	|..|
|GD	|0.3	|0.3	|0.31|
|PIC	|0.24	|0.25	|0.25|
|MAF	|0.21	|0.22	|0.22|
|Ho	|0.03	|0.03	|0.03|
|F	|0.89	|0.91	|0.9|

--
#### F-statistics 
|Pop	|Fis	|Fst|	Fit|
|-----|----|------|-------|
|Esalq x Iapar|	0.901|	0.021|	0.902|


---
#Outlook

- Hyperspectral camera


--


- Random regression from high-throughput phenotyping data with Association Weight Matrix
---
&lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt; &lt;br&gt;
# .center[Thank you] 

###.right[ Rafael Massahiro Yassue ]
.right[![](qr.png)]
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/remark/0.14.0/remark.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
